app/assets/stylesheets/redesign/_global.scss

```diff
@@ -0,0 +1,55 @@
+// Colors
+$c2blue-swatch: (
```

Comment:
Did you see the variable file? @micahsaul 

https://github.com/18F/C2/blob/master/app/assets/stylesheets/redesign/_foundation_settings.scss